src/tracing/errors.ts

```diff
@@ -8,10 +8,19 @@ declare global {
   }
 }
 
-export function beforeSend(event: ErrorEvent, hint: EventHint) {
+/**
+ * Filters known (ignorable) errors out before sending them to Sentry. Also, adds tags to the event.
+ * Intended as a {@link ClientOptions.beforeSend} callback. Returning null filters the error from Sentry.
+ */
+export const beforeSend: Required<ClientOptions>['beforeSend'] = (event: ErrorEvent, hint: EventHint) => {
```

Comment:
This is an exceptional case where I prefer `const fn = () => {` over `function fn() {` because you can then apply the derived type, as you do :)